services/api/.gitignore

```diff
@@ -1,2 +1,3 @@
+.env
```

Comment:
Why `.env` file is excluded from source control?

We should not commit any sensitive environment configs to git, but it would be great to have `.env` file with configuration for local environment when checking out repository for the first time as it reduces time need for setting up local dev environment.

Reply:
I think we should NOT commit .env. Instead we should have .env.example for defaults. Thats a well know practice for .env files.

Reply:
No problem with calling it `.env.example` or any other name, as long as it is clear what are the expected values for local development.